bugbug/db.py

```diff
@@ -119,12 +119,14 @@ def upload(path):
 
 
 def last_modified(path):
-    url = DATABASES[path]["url"]
-    last_modified = utils.get_last_modified(url)
 
-    if last_modified is None:
+    if is_different_schema(path):
         raise LastModifiedNotAvailable()
 
+    url = DATABASES[path]["url"]
+    last_modified = utils.get_last_modified(url)
+    # if no Last-Modified header then that needs to be handled. (in another PR)
```

Comment:
Please keep the same behavior as before, don't change it. The test can be added in a separate PR, but the behavior here should not change

Reply:
Ok